### PR TITLE
Update Sudo mode ...(Grammatical Mistakes etc.)

### PR DESCRIPTION
**Pull request by Eldrick99**
### Why:
It makes the world a  better place

### What's being changed:
Link - https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/sudo-mode
I fixed a grammatical mistake (Its THE TIMER not this timer) ...
I updated an image ..
I made the wikipedia example link https In the code..(It was http)
And I added an Example (Delete a repository)

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by Eldrick99** [comment]
Someone please Approve My changes

**Comment by Eldrick99** [comment]
### OLD IMAGE

### NEW IMAGE

**Comment by ramyaparimi** [comment]
@Eldrick99 Thank you for contributing to the docs, unfortunately these changes are not something the docs is looking to update. I am going to close this PR now :yellow_heart: 
Take a look at the [contributing.md](https://github.com/github/docs/blob/main/CONTRIBUTING.md/) to learn how to contribute in this repo. You can open a [discussion](https://github.com/github/docs/discussions) if you have any questions :sparkles:

**Comment by Eldrick99** [comment]
Ok fine, I'll not contribute to the docs ..
I tried many times.

**Comment by Eldrick99** [comment]
#Ramya
Its not only about the image I also fixed a typo.......
Can you please check again....

File: content/github/authenticating-to-github/keeping-your-account-and-data-secure/sudo-mode.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ topics:
   - Identity
   - Access management
 ---
-After you've performed a sudo-protected action, you'll only be asked to re-authenticate again after a few hours of inactivity. Every sudo-protected action resets this timer.
+After you've performed a sudo-protected action, you'll only be asked to re-authenticate again after a few hours of inactivity. Every sudo-protected action resets the timer.
```
Comment (Eldrick99) [inline content/github/authenticating-to-github/keeping-your-account-and-data-secure/sudo-mode.md:14]: See This

File: content/github/authenticating-to-github/keeping-your-account-and-data-secure/sudo-mode.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Sudo mode
-intro: '{% data variables.product.product_name %} asks you for your password before you can modify your email address, authorize third-party applications, or add new public keys, or initiate other *sudo-protected* actions.'
+intro: '{% data variables.product.product_name %} asks you for your password before you can modify your email address, delete a repository, authorize third-party applications, or add new public keys, or initiate other *sudo-protected* actions.'
```
Comment (Eldrick99) [inline content/github/authenticating-to-github/keeping-your-account-and-data-secure/sudo-mode.md:3]: @ramyaparimi I also added an example

**Pull request closed without merging**